app/initializers/raven-setup.js

```diff
@@ -14,10 +14,17 @@ export function initialize() {
     throw new Error('`sentry` should be configured when not in development mode.');
   }
 
-  Raven.config(config.sentry.dsn, {
-    release: config.APP.version,
-    whitelistUrls: config.sentry.whitelistUrls
-  }).install();
+  var options = {};
+
+  options.whitelistUrls = config.sentry.whitelistUrls;
+
+  options.release = config.APP.version;
+
+  if (typeof config.sentry.dataCallback === 'function') {
```

Comment:
This is where we want to check service existence.
If it exist (we can assume this is a function for our needs), then data callback is that service.

This also makes the parser much more testable by the way.